CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,118 @@
+  backgrounds and identities. This includes, but is not limited to people of
+  any origin, color, status, educational level, gender identity, sexual
+  orientation, age, culture and beliefs, and mental and physical ability.
+- **Be considerate.**
+  Your work will be used by other people, and you in turn will depend on the
+  work of others. Any decision you take will affect users and colleagues, and
+  you should take those consequences into account when making decisions.
+- **Be respectful.**
+  Not all of us will agree all the time, but disagreement is no excuse for poor
+  behavior and poor manners. We might all experience some frustration now and
+  then, but we cannot allow that frustration to turn into a personal attack.
+  It's important to remember that a community where people feel uncomfortable
+  or threatened is not a productive one. Members of the Coq development team
+  and user community should be respectful when dealing with other members as
+  well as with people outside the community.
+- **Be careful in the words that you choose.**
+  Be kind to others. Do not insult or put down other participants. Harassment
+  and other exclusionary behavior aren't acceptable.
+  * Violent language or threats or personal insults have no chance to
+    resolve a dispute or to let a discussion florish. Worse, they can
+    hurt durably, or generate durable fears. They are thus unwelcome.
+  * Not everyone is comfortable with sexually explicit or violent
+    material, even as a joke. In an online open multicultural world, you
+    don't know who might be listening. So be cautious and responsible
+    with your words.
+  * Discussions are online and recorded for posterity; we all have our
+    right for privacy and online gossiping as well as posting or threatening to
+    post other people's personally identifying information is prohibited.
+- **Remember that what you write in a public online forum might be read by
+  many people you don't know.**
+  Consider what image your words will give to outsiders of the development
+  team / the user community as a whole. Try to avoid references to private
+  knowledge to be understandable by anyone.
+- **Coq online forums are only to discuss Coq-related subjects.**
+  Unrelated political discussions or long digressions are unwelcome,
+  even for illustration or comparison purposes.
+- **When we disagree, try to understand why.**
+  Disagreements, both social and technical, happen all the time and Coq is no
+  exception. It is important that we resolve disagreements and differing views
+  constructively. Remember that we are different. Different people
+  have different perspectives on issues. Being unable to understand why someone
+  holds a viewpoint doesn't mean that they're wrong.
+- **It is human to make errors, and please try not to take things personally.**
+  Please do not to answer aggressively to problematic behavior and simply
```

Comment:
Please do not answer

Reply:
I think this changes the meaning. @CohenCyril This is your contributed paragraph. WDYT?

Reply:
I think that's a typo (I think I hesitated between "do" and "try") and it should be fixed: "Please do not answer aggressively to [...]"

Reply:
Ah sorry, I had misunderstood the change @ybertot that was proposing. Will fix the typo.